### PR TITLE
Basic CSR React chat example 

### PR DESCRIPTION
**Pull request by juberti**
This is a basic chat example, cut down from the existing CYOA example, that is intended to be a good starting point for various conversational apps. Support for docsqa will be added in a similar example in an upcoming PR.

### CODE REVIEW

File: packages/create-react-app-demo/src/basic-chat/ai.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+// @ts-nocheck
```
Comment (NickHeiner) [inline packages/create-react-app-demo/src/basic-chat/ai.tsx:1]: This is no longer necessary after #51 landed.

**Comment by NickHeiner** [comment]
Damn, gpt-3.5-turbo feels shockingly fast as I play around with this.

**Comment by NickHeiner** [comment]
Looks great! I think someday we'll have richer abstractions for devs to use in a few places here.

**Review by NickHeiner — APPROVED** [review]

**Comment by juberti** [comment]
agreed, I want to build a history component that should make things a bit simpler

**Pull request merged**